ui/core/src/ui-core/autosuggest/AutoSuggest.tsx

```diff
@@ -64,7 +64,7 @@ export interface AutoSuggestProps extends React.InputHTMLAttributes<HTMLInputEle
   /** Use it only if you need to customize the rendering of the input.
    * @internal
    */
-  renderInputComponent?: ReactAutosuggest.RenderInputComponent<AutoSuggestData>;
+  renderInputComponent?: any; // ReactAutosuggest.RenderInputComponent<AutoSuggestData>; NOTE: changed to "any" because build was failing with RenderInputComponent is not generic?

```

Comment:
@DanEastBentley I assume that there is a better fix. Not sure why this was failing the CI build since I didn't change anything in `ui`.

Reply:
In pnpm-lock.yaml, what is the version of @types/react-autosuggest? Is it 10.1.1 or 10.1.2? If you right-click and go to the definition of RenderInputComponent, does it look like this:
type RenderInputComponent<TSuggestion> = (inputProps: InputProps<TSuggestion>) => React.ReactNode;

Reply:
`10.1.2` and `type RenderInputComponent = (inputProps: RenderInputComponentProps) => React.ReactNode`

Reply:
Leave it as 'any' for now and I'll look into it after your PR.